### PR TITLE
🐛 schema: missing values

### PR DESCRIPTION
**Pull request by lucasrodes**
- `baseColorScheme`: "OwidDistinctLines"

### CODE REVIEW

**Comment by marcelgerber** [comment]
@danyx23 do you know off the top of your head if we're also missing other color schemes here that can go there, like `OwidCategoricalA` etc.?

File: packages/@ourworldindata/grapher/src/schema/grapher-schema.003.yaml [hunk 1]
```diff
@@ -103,6 +103,7 @@ $defs:
                     - stackedAreaDefault
                     - owid-distinct
                     - default
+                    - OwidDistinctLines
```
Comment (danyx23) [inline packages/@ourworldindata/grapher/src/schema/grapher-schema.003.yaml:106]: To complete all new color values I'd suggest this (OwidDistinctLines is included in the middle):
```suggestion
                    - ContinentsLines
                    - SingleColorDenim
                    - SingleColorTeal
                    - SingleColorPurple
                    - SingleColorDustyCoral
                    - SingleColorDarkCopper
                    - OwidCategoricalA
                    - OwidCategoricalB
                    - OwidCategoricalC
                    - OwidCategoricalD
                    - OwidCategoricalE
                    - OwidEnergy
                    - OwidEnergyLines
                    - OwidDistinctLines
                    - BinaryMapPaletteA
                    - BinaryMapPaletteB
                    - BinaryMapPaletteC
                    - BinaryMapPaletteD
                    - BinaryMapPaletteE
                    - SingleColorGradientDenim
                    - SingleColorGradientTeal
                    - SingleColorGradientPurple
                    - SingleColorGradientDustyCoral
                    - SingleColorGradientDarkCopper
```

File: packages/@ourworldindata/grapher/src/schema/grapher-schema.003.json [hunk 2]
```diff
@@ -109,7 +109,8 @@
                         "continents",
                         "stackedAreaDefault",
                         "owid-distinct",
-                        "default"
+                        "default",
+                        "OwidDistinctLines"
```
Comment (danyx23) [inline packages/@ourworldindata/grapher/src/schema/grapher-schema.003.json:113]:
```suggestion
                        "ContinentsLines",
                        "SingleColorDenim",
                        "SingleColorTeal",
                        "SingleColorPurple",
                        "SingleColorDustyCoral",
                        "SingleColorDarkCopper",
                        "OwidCategoricalA",
                        "OwidCategoricalB",
                        "OwidCategoricalC",
                        "OwidCategoricalD",
                        "OwidCategoricalE",
                        "OwidEnergy",
                        "OwidEnergyLines",
                        "OwidDistinctLines",
                        "BinaryMapPaletteA",
                        "BinaryMapPaletteB",
                        "BinaryMapPaletteC",
                        "BinaryMapPaletteD",
                        "BinaryMapPaletteE",
                        "SingleColorGradientDenim",
                        "SingleColorGradientTeal",
                        "SingleColorGradientPurple",
                        "SingleColorGradientDustyCoral",
                        "SingleColorGradientDarkCopper"
```

**Review by danyx23 — APPROVED** [review]
@marcelgerber good idea - this here is the updated complete list from our type definitions. Thanks for doing this, @lucasrodes!

**Pull request merged**